### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by balaharish7**

### CODE REVIEW

**Comment by balaharish7** [comment]
Hi @avishnu, Kindly review and verify if the PR is good to merge.

File: docs/main/user-guides/upgrades.md [hunk 1]
```diff
@@ -5,156 +5,127 @@ keywords:
...
-### Identifying the helm chart
+This upgrade flow would allow users to upgrade to the latest OpenEBS version 4.0.0 which is a unified installer for three Local Storages (a.k.a Local Engines) Local PV HostPath, Local PV LVM, Local PV ZFS, and one Replicated Storage (a.k.a Replicated Engine and f.k.a Mayastor). 
+As a part of upgrade to OpenEBS 4.0.0, the helm chart would install all four engines irrespective of the engine the user was using prior to the upgrade. 
 
-You'll need the helm v3 binary to be on your $PATH. Execute the following command to check if you have helm v3:
-```
-$ helm version --short
-v3.9.4+gdbc6d8e
-```
+During the upgrade, advanced users who are only interested in Local PV Storage, can disable the Replicated Storage by setting the below option:
```
Comment (avishnu) [inline docs/main/user-guides/upgrades.md:27]: Should we make the above line as a note or highlighted section?

Comment (balaharish7) [reply docs/main/user-guides/upgrades.md:27]: We can give this in an info section

File: docs/main/user-guides/upgrades.md [hunk 2]
```diff
@@ -5,156 +5,127 @@ keywords:
...
-:::note
-It is a good idea to throw in the `--atomic` helm upgrade flag, so that the cluster may be reinstated to its source version, in case the upgrade to the target version fails. E.g.:
 ```
-helm upgrade openebs-mayastor openebs/openebs -n openebs --version 3.8.0 --set mayastor.image.tag=$IMG_TAG --set release.version=3.8.0 --reuse-values --atomic
+kubectl mayastor upgrade -n openebs
 ```
-:::
 
-#### 3. Execute the helm upgrade command
+- The upgrade should be deployed.
```
Comment (avishnu) [inline docs/main/user-guides/upgrades.md:108]: This deploys an upgrade process of K8s resource type Job

File: docs/main/user-guides/upgrades.md [hunk 3]
```diff
@@ -5,156 +5,127 @@ keywords:
...
+```
+kubectl get jobs -n openebs 
 
-#### 4. Follow the upgrade instructions for the mayastor helm chart
+NAME                     COMPLETIONS   DURATION   AGE 
+openebs-upgrade-v2-6-0   1/1           4m49s      6m11s
+```
 
-Follow the upgrade instruction for the mayastor helm chart -> [https://mayastor.gitbook.io/introduction/additional-information/upgrade](https://mayastor.gitbook.io/introduction/additional-information/upgrade).
+- Wait for the upgrade to get complete.
```
Comment (avishnu) [inline docs/main/user-guides/upgrades.md:117]: Wait for the upgrade job to complete.

**Review by avishnu — APPROVED** [review]
lgtm

**Pull request merged**